Objects/Spawners/DevilSpawner/DevilSpawner.gd

```diff
@@ -6,4 +6,7 @@ onready var world = get_node("/root/World")
 
 func spawn_devil():
 	var devil = create_object(Devil, world)
-	devil.global_position = global_position
\ No newline at end of file
+
+	var cam_pos = get_node("/root/World").find_node("Camera2D").position
+	devil.position.x = cam_pos.x + get_viewport_rect().size.x
+	devil.position.y = 280
```

Comment:
280 is const. You can create variable `const TRACK_HEIGHT = 280` and 
change this line on `devil.position.y = TRACK_HEIGHT ` 
This variable should be in `SpawnerSystem`

Reply:
Done